denorm/formats/join.json

```diff
@@ -74,151 +97,198 @@
           "description": "Unique key for foreign table.",
           "items": {
             "type": "string"
           },
           "title": "Join key",
-          "type": ["array", "null"]
+          "type": [
+            "array",
+            "null"
+          ]
+        },
+        "joinOn": {
+          "default": null,
+          "description": "SQL expression to join to dependency.",
+          "title": "Dependency join",
+          "type": [
+            "string",
+            "null"
+          ]
         },
         "joinMode": {
           "default": "sync",
-          "description": "Mode of dependency join.",
-          "enum": ["async", "sync"],
+          "description": "Mode of dependency join. Large many to one should use 'async'",
+          "enum": [
+            "async",
+            "sync"
+          ],
           "title": "Join consistency",
-          "type": ["string", "null"]
+          "type": [
+            "string",
+            "null"
+          ]
         },
         "joinOther": {
           "default": null,
           "description": "Expressions to add to join.",
           "title": "Join other",
-          "type": ["string", "null"]
-        },
-        "joinOn": {
-          "default": null,
-          "description": "SQL expression to join to dependency.",
-          "title": "Dependency join",
-          "type": ["string", "null"]
-        },
-        "key": {
-          "default": null,
-          "description": "Unique key.",
-          "items": {
-            "$ref": "#/definitions/keyColumn"
-          },
-          "title": "key",
-          "type": ["array", "null"]
-        },
-        "name": {
-          "description": "Name of table.",
-          "title": "Name",
-          "type": "string"
+          "type": [
+            "string",
+            "null"
+          ]
         },
         "refreshFunction": {
           "default": false,
           "description": "Whether to generate a refresh function.",
           "title": "Refresh function",
           "type": "boolean"
         },
-        "schema": {
-          "default": "None",
-          "description": "Name of schema.",
-          "title": "Schema",
-          "type": "string"
-        },
-        "targetKey": {
+        "destinationKeyExpr": {
           "default": null,
-          "description": "SQL expressions for target key values.",
+          "description": "SQL expressions for this table's columns that make up the destination table's key columns ([this table name].[column name]).",
           "items": {
             "type": "string"
           },
           "title": "key",
-          "type": ["array", "null"]
+          "type": [
+            "array",
+            "null"
+          ]
         }
       },
       "type": "object",
       "title": "Table"
     },
-    "targetTable": {
-      "description": "Target table.",
+    "destinationTable": {
+      "description": "Destination table where denormalized data will be stored.",
```

Comment:
I think this makes much more sense to me. We have a `table` that joins to the `targetTable` and the denormalized data is put in a `destinationTable` if I have been reading this correctly.